src/utils.ts

```diff
@@ -17,6 +17,169 @@ const isObject = (value: unknown) =>
   !(value instanceof Error) &&
   !(value instanceof Date);
 
+// toCidr takes an IP and prefix length and returns a normalized CIDR.
+export function toCidr(ip: string, prefixLen: number) {
```

Comment:
Would we want to handle the case when `ip` is a non-IP string?  It probably won't happen, I suppose.

Reply:
Did you have something in mind? I think `mmdb-lib` would have already thrown an error if something other than a string was passed as the `ip`.

Reply:
We also do [this](https://github.com/gohugoio/hugo/blob/5b0b663ec321454c747fd58a4d0d8cc92e32a746/tpl/tplimpl/embedded/templates/opengraph.html#L11-L13). It is kind of unfortunate that the IPs are getting parsed at least three times, but it probably doesn't matter too much.

Reply:
If `mmdb-lib` is already validation, we should be fine then.  Also, the link you posted goes to some hugo documentation....

Reply:
Gah, multitasking. I updated [the link](https://github.com/maxmind/GeoIP2-node/blob/2aef8547abec5818d13790f19dd8455b3c780c12/src/readerModel.ts#L143).